org_fedora_hello_world/tui/spokes/hello_world.py

```diff
@@ -198,84 +193,20 @@ def input(self, args, key):
 
         # close the current screen (remove it from the stack)
         self.close()
-        return True
+        return InputState.PROCESSED
 
     def prompt(self, args=None):
         """
         The prompt method that is called by the main loop to get the prompt
         for this screen.
 
-        :see: pyanconda.ui.tui.simpleline.Prompt
+        :see: simpleline.render.prompt.Prompt
 
         :param args: optional argument that can be passed to App.switch_screen*
                      methods
         :type args: anything
         :return: text that should be used in the prompt for the input
-        :rtype: instance of pyanconda.ui.tui.simpleline.Prompt or None
+        :rtype: instance of simpleline.render.prompt.Prompt or None
         """
 
         return Prompt(_("Enter a new text or leave empty to use the old one"))
-
-class _EditData(object):
-    """Auxiliary class for storing data from the example EditSpoke"""
-
-    def __init__(self):
-        """Trivial constructor just defining the fields that will store data"""
-
-        self.checked = False
-        self.shown_input = ""
-        self.hidden_input = ""
-
-class HelloWorldEditSpoke(EditTUISpoke):
-    """Example class demonstrating usage of EditTUISpoke inheritance"""
-
-    title = N_("Hello World Edit")
-    category = HelloWorldCategory
-
-    # simple RE used to specify we only accept a single word as a valid input
-    _valid_input = re.compile(r'^\w+$')
-
-    # special class attribute defining spoke's entries as:
-    # Entry(TITLE, ATTRIBUTE, CHECKING_RE or TYPE, SHOW_FUNC or SHOW)
-    # where:
-    #   TITLE specifies descriptive title of the entry
-    #   ATTRIBUTE specifies attribute of self.args that should be set to the
-    #             value entered by the user (may contain dots, i.e. may specify
-    #             a deep attribute)
-    #   CHECKING_RE specifies compiled RE used for deciding about
-    #               accepting/rejecting user's input
-    #   TYPE may be one of EditTUISpoke.CHECK or EditTUISpoke.PASSWORD used
-    #        instead of CHECKING_RE for simple checkboxes or password entries,
-    #        respectively
-    #   SHOW_FUNC is a function taking self and self.args and returning True or
-    #             False indicating whether the entry should be shown or not
-    #   SHOW is a boolean value that may be used instead of the SHOW_FUNC
-    #
-    #   :see: pyanaconda.ui.tui.spokes.EditTUISpoke
-    edit_fields = [
-        Entry("Simple checkbox", "checked", EditTUISpoke.CHECK, True),
-        Entry("Always shown input", "shown_input", _valid_input, True),
-        Entry("Conditioned input", "hidden_input", _valid_input,
-              lambda self, args: bool(args.shown_input)),
-        ]
```

Comment:
I believe we still need an example of such "inputs".

Reply:
Ok, then I'll add them back but I need to do it with Simpleline.
Right now kdump addon has a priority so I'll fix this later.

Reply:
On the other hands this will be changed dramatically because of the Anaconda modularization effort so I think we would need to change the hello world later even more. In this case I don't see a point in improving it right now. 

There's a link on the Simpleline documentation where is this described already. I know it is not that comfortable but it should be enough.

Reply:
> Ok, then I'll add them back but I need to do it with Simpleline.

People having their own addons or writing new addons will have to to this too. Don't take me wrong, but if it's really that hard for you as the author of Simpleline, how can you expect people being able to do it?

Reply:
It's not that hard but it is unpleasant and time consuming to do it in Anaconda by updates image. And yes, Simpleline still has reserve in comfort of usage and I hope I'll be able to fix these soon. I'll already started. One new simplification is on it's way and another is in TODO list. BTW I would welcome any ideas for improvement as issues in repo or bugs in bugzilla just in case you have some.

Ok, I guess I can take example solution and just polite it, then it shouldn't take much time.